addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -63,12 +64,12 @@ def format_monetary(number, currency):
             'format_monetary': format_monetary,
         }
 
-        xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
+        xml_content = markupsafe.Markup("<?xml version='1.0' encoding='UTF-8'?>")
         xml_content += self.env.ref('account_edi_facturx.account_invoice_facturx_export')._render(template_values)
         xml_name = '%s_facturx.xml' % (invoice.name.replace('/', '_'))
         return self.env['ir.attachment'].create({
             'name': xml_name,
-            'datas': base64.encodebytes(xml_content),
+            'datas': base64.encodebytes(xml_content.encode('utf-8')),
```

Comment:
we can have the same simplification everywhere I suppose (no need to b64 encode the data if we need to decode it immediately to store it on disk) ;-)

```suggestion
            'raw': xml_content.encode(),
```